features/object_mapper/creating/README.md

```diff
@@ -0,0 +1,168 @@
+## Definition of mapped classes
+
+The object mapper allows to define mapped classes by using annotations.
+ 
+### Creating a table entity
+ 
+A table entity is generally meant to match the definition of an existing table in the *Cassandra* server.
```

Comment:
I was making a general statement because it is not always the case, people can use `@Transient` or `@Computed` and then the `@Table` class differs from the table in the database